service/src/main/java/uk/gov/hmcts/reform/fpl/model/ReturnApplication.java

```diff
@@ -17,4 +19,15 @@
     private String submittedDate;
     private String returnedDate;
     private DocumentReference document;
+
+    @JsonIgnore
+    public String getFormattedReturnReasons() {
+        if (reason != null) {
+            return reason.stream()
+                .map(ReturnedApplicationReasons::getLabel)
+                .collect(Collectors.joining(", "));
```

Comment:
Going to be super picky here but this means that the output will be along the lines of "Application Incomplete, Clarification Needed, etc", are we sure that we want it capitalised like that in the email?

Reply:
Nice spot yeah I agree let's fix that.